### PR TITLE
Homework5: Points and Lines

### PR DESCRIPTION
**Pull request by MagaHovhannisyan**
Smbat jan, please find the homework related to Points and Lines. I will add TestNG cases started from Homework6.

### CODE REVIEW

File: src/class5PointsAndLines/Line.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package class5PointsAndLines;
+
+import java.text.DecimalFormat;
+
+public class Line {
+    private Point a;
+    private Point b;
+    private static double length;
```
Comment (Smbat888) [inline src/class5PointsAndLines/Line.java:8]: length-y inchia static ?

mincchev Line-ic object chunenas length ches kara unenas, 

isk ete static es dnum, nshanakuma vor Line-i objectic kaxvacc chi, ayl Line class-ic a kaxvac (aysinqn bolor objectneri hamar kopit asac et mi lengthna linelu), esi poxi) voch static dir

File: src/class5PointsAndLines/PointLineDemo.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        Point pointC = new Point(20, 20);
+        Point pointD = new Point(10, 20);
+        Point pointE = new Point(11, 11);
+
+        /*
+         * 1. Calculate the distance between two points.
+         */
+        System.out.println("1. Here is the distance between the two points : ");
+        Line line = new Line(point1, point2);
+        Line.getLength();
```
Comment (Smbat888) [inline src/class5PointsAndLines/PointLineDemo.java:18]: esi vor static-ic hanes, arden kkanches objecti vra voch te class-i
Line line = new Line(point1, point2);
double length = line.getLength();

File: src/class5PointsAndLines/API.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package class5PointsAndLines;
+
+public class API {
+    private static double middleA;
```
Comment (Smbat888) [inline src/class5PointsAndLines/API.java:4]: middleA u middleBB, sranc static sarqeln el chhaskaca te xi))
esi senc sxala eli) nuyn nerqevi asacs patcharov

**Review by Smbat888 — COMMENTED** [review]
apres, lava,
menak mi ban, static-neri het kapvac nkatum em vor poqr xndirner unes, mi hat petqa sra vrayov ancnenq irar het vor imanam vor lav patkeracnum es ))

**Comment by MagaHovhannisyan** [comment]
> apres, lava,
> menak mi ban, static-neri het kapvac nkatum em vor poqr xndirner unes, mi hat petqa sra vrayov ancnenq irar het vor imanam vor lav patkeracnum es ))

Ha, mitesak inch tesnum em uzum em static dnem :D
Irakanum es verjin temanery miqich uxexums dasavorelu kariq ka, mihat el vrayov kancnem, kpordzem aveli hstak haskanal.

File: src/class5PointsAndLines/API.java (hunk 3 above)
Comment (MagaHovhannisyan) [reply src/class5PointsAndLines/API.java:4]: Merci shat commentneri hamar, staticneric mi 2 ban avel imaca)) bolor popoxutyunnern arel em, arden merge kanem PR-y))

**Pull request merged**